components/Navbar.vue

```diff
@@ -264,12 +264,14 @@
 <script>
 import website from '~/static/website.json';
 import social from '~/static/social-networks.json';
+import bureauxJSON from '~/static/bureaux.json';
 
 export default {
   name: 'Navbar',
   data() {
     return {
       hdr: website,
+      bureau: bureauxJSON,
```

Comment:
Consider removing the `bureaux` attribute in `~/static/bureaux.json` and just put the data directly to avoid using `bureauxJSON.bureaux` and just use `bureauxJSON`.

```
{
  "bureaux":[{emoji:"",nom:"",...}]
}
```

To :

```
[{emoji:"",nom:"",...}]
```